src/store/index.js

```diff
@@ -1,10 +1,22 @@
 import Vue from "vue";
 import Vuex from "vuex";
+import movies from "../assets/movies.json";
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  state: {},
+  state: {
+    movies,
```

Comment:
Consider creating "getters" file under "src/store" to keep getters names as variables there.
By doing so you can utilise dynamic object keys as well as make sure you're using the same getters name across the app.
E.g `...., state: { [GETTERS.MOVIES]: movies}, getters: { [GETTERS.MOVIES_COUNT]: function(...) {...}, }, ...`
or `...mapState([GETTERS.MOVIES])`

Reply:
Fixed